ansible/roles/stack-sunbird/templates/orchestrator.env

```diff
@@ -31,7 +31,7 @@ FORMS_DB_PORT=5432
 CASSANDRA_URL={{sunbird_cassandra_host}}
 CASSANDRA_PORT=9042
 CASSANDRA_KEYSPACE={{env}}_uci_store
-CASSANDRA_MIGRATION_COUNT=1
+CASSANDRA_MIGRATION_COUNT={{uci_cassandra_migration_count}}
```

Comment:
@surabhi-mahawar Please add a default value also. Something like this
`CASSANDRA_MIGRATION_COUNT={{uci_cassandra_migration_count | default(1)}}`